src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -49,30 +48,46 @@ const otherToolsLink = (
   </p>
 );
 
+const headingStyle = {
+  fontWeight: '700',
+  fontFamily: 'Bitter, Georgia, Cambria, Times New Roman, Times, serif',
+  lineHeight: '1.3',
+  clear: 'both',
+};
+
+const ddStyle = {
+  margin: '2rem 0 .5rem 0',
+  lineHeight: '1.5',
+};
```

Comment:
There are utilities for margin and line-height too - https://design.va.gov/utilities/line-height